jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -377,6 +409,24 @@ export class MountPlugin implements IMountPlugin {
     this.open('');
   };
 
+  setShowPipeline = (shouldShow: boolean): void => {
+    if (shouldShow) {
+      this._pipeline.setHidden(false);
+      this._mountedList.setHidden(true);
+      this._unmountedList.setHidden(true);
+    } else {
+      this._pipeline.setHidden(true);
+      this._mountedList.setHidden(false);
+      this._unmountedList.setHidden(false);
+    }
+    this._mountBrowser.setHidden(false);
```

Comment:
We don't need the file browser in the bottom of the extension when we're in pipeline mode right? If not, we can hide it to give more space to the pipeline UI.

Reply:
Nice I was wondering how to get rid of that, haha